addons/l10n_es_pos/static/src/overrides/models/models.js

```diff
@@ -0,0 +1,70 @@
+/** @odoo-module */
+import { patch } from "@web/core/utils/patch";
+import { Order } from "@point_of_sale/app/store/models";
+import { isConnectionError } from "@point_of_sale/utils";
+
+patch(Order.prototype, "l10n_es_pos.Order", {
+    set_simple_inv_number() {
+        return this.pos
+            .get_simple_inv_next_number()
```

Comment:
As you await the call to the function set_simple_inv_number, you could await the call to get_simple_inv_next_number and rewrite the code as a classical try catch (more adequate with the way the PoS code is written) ? What do you think ?

Reply:
done!